lib/slack/bot.ex

```diff
@@ -87,11 +105,11 @@ defmodule Slack.Bot do
       token: token,
       me: rtm.self,
       team: rtm.team,
-      bots: rtm_list_to_map(rtm.bots),
-      channels: rtm_list_to_map(rtm.channels),
-      groups: rtm_list_to_map(rtm.groups),
-      users: rtm_list_to_map(rtm.users),
-      ims: rtm_list_to_map(rtm.ims)
+      bots: bot_to_map(Bots.info(%{token: token}) |> Map.get("bot")),
+      channels: rtm_list_to_map(Channels.list(%{token: token}) |> Map.get("channels")),
+      groups: rtm_list_to_map(Groups.list(%{token: token}) |> Map.get("groups")),
+      users: rtm_list_to_map(Users.list(%{token: token}) |> Map.get("members")),
+      ims: rtm_list_to_map(Im.list(%{token: token}) |> Map.get("ims"))
```

Comment:
All these lists are likely partial responses. It's better to not include this info at all instead of trying to preserve a semblance of backwards compatibility. Especially considering that old `rtm.start` is truncating members lists to 500 records for almost a year already.

Reply:
> All these lists are likely partial responses.

So if it's at parity with `rtm.start` then are we losing any more functionality than we already had?

Reply:
Good point. I would still make it optional personally. The data in these keys is definitely not immutable on Slack side and it's best to fetch, cache, and expire it as specific bot requires.